src/acom_music_box/conditions.py

```diff
@@ -179,8 +179,13 @@ def from_config_JSON(
         for reaction in reaction_list.reactions:
             if (reaction.name is None):
                 continue
-            if not any(rate.reaction.name ==
-                       reaction.name for rate in reaction_rates):
+            reaction_exists = False
+            for rate in reaction_rates:
+                if rate.reaction.name == reaction.name:
+                    reaction_exists = True
+                    break
+
+            if not reaction_exists:
```

Comment:
Don't actually know what this does but it's apparently crucial and tests fail without it